modules/monitoring-configuring-audit-logs-for-metrics-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="configuring-audit-logs-for-metrics-server_{context}"]
+= Configuring audit logs for Metrics Server
+
+You can configure audit logs for Metrics Server to help you troubleshoot issues with the server. 
+Audit logs record the sequence of actions in a cluster. It can record user, application, or control plane activities.
+
+You can set audit log rules, which determine what events are recorded and what data they should include. This can be achieved with the following audit profiles:
+
+* *Metadata (default)*: This profile enables the logging of event metadata (user, timestamps, resource, verb, and others). It does not record request and response bodies.
+* *Request*: This enables the logging of event metadata and request body, but it does not record response body. This configuration does not apply for non-resource requests.
+* *RequestResponse*: This enables the logging of event metadata, request and response bodies. This configuration does not apply for non-resource requests.
```

Comment:
I wasn't sure if "request and response bodies" is one thing or two. So I think this sentence should either be "...logging of event metadata, request, and response bodies." or "...logging of event metadata, and request and response bodies."